app/views/partials/foyer/ressources/types.html

```diff
@@ -14,22 +14,23 @@
             is-open="status.open"
             ng-init="status.open = shouldInitiallyOpen(category)">
             <uib-accordion-heading>
-              {{ category.label }}
-              <i class="pull-left fa" aria-hidden="true"
+              <i class="pull-left fa"
```

Comment:
No, and it does actually improve rendering to remove it!

Current:
![capture d ecran 2016-08-12 a 14 40 47](https://cloud.githubusercontent.com/assets/222463/17622461/d224c75e-609a-11e6-9712-a845ce8a6fb5.png)

Without `pull-left`:
![capture d ecran 2016-08-12 a 14 40 55](https://cloud.githubusercontent.com/assets/222463/17622468/daa640e2-609a-11e6-8372-c47f1ee2a409.png)